docs/jwst/pipeline/calwebb_dark.rst

```diff
@@ -32,8 +32,8 @@ pipeline, the order of steps is a bit different for MIRI exposures.
 |                                       | :ref:`rscd <rscd_step>`                 |
 +---------------------------------------+-----------------------------------------+
 
-.. [1] The :ref:`ipc <ipc_step>` step is currently set to be skipped by default in the
-   "calwebb_dark.cfg" configuration file for all instruments.
+.. [1] By default, the step parameter reference `pars-darkpipeline`
```

Comment:
Could probably remove the word "step" here, because in this particular example `pars-darkpipeline` is a param file for a pipeline, not a step. I know that we developers know that a pipeline inherits from `Step` and hence in terms of python objects they're kind of the same thing, but your average Joe user isn't going to or want to know that.